### ISSUE TITLE
[RFC]: Adapt scaffold to support new Byte format instead of [object object] rendering

### ISSUE THREAD

**Issue by matbgn** [redwoodjs/redwood#10288]
### Summary

Take into consideration that Bytes are now supported and should be treated accordingly on Form generated by scaffold like on the screenshot below:

![image](https://github.com/redwoodjs/redwood/assets/13169819/c2ce0179-11a2-4981-992f-2cfbd1b9fa49)

The aim is that `yarn rw g scaffold <...>` will deal with a really basic mechanic of downloading and uploading bytes files instead of [object object] rendering.

### Motivation

Based on feedback from @dthyresson I submit this RFC 

https://community.redwoodjs.com/t/redwood-v7-0-0-upgrade-guide/5713/6

### Detailed proposal

Basically, I would propose to adapt the scaffold CLI to use FileField instead of TextField within the CRUD Form for the uploading part. 

For the rendering part, I would just say that a direct download is totally acceptable for a first step or maybe display it’s Base64/Raw equivalent like the dbStudio does.

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by matbgn** [comment]
I wasn't able to eject it properly, possibly due to misuse. Anyway, I suggest just downloading the byte and not trying to display it. This means that the contents of the byte must be displayed with a download logo.
